docs/getting-started/plugins.md

```diff
@@ -15,14 +15,42 @@ The installed plugins are shown on the _Installed Plugins_ screen.
 
 ## Available Plugins
 
-The following plugins are currently available, we are working to add more plugins 
-
-- MongoDB 
-- PostgreSQL DB
-- MySQL DB
-- Kafka
-- Password JWT Authentication
-- Password Basic Authentication
+The following plugins are currently available. Addition plugins will be added in future releases of Amplication.  
+
+#### MongoDB
+
+[MongoDB](https://www.mongodb.com/) is a free and open-source cross-platform document-oriented database program. It is classified as a NoSQL database program, meaning that it does not use the traditional SQL relational database management system. Instead, it uses JSON-like documents with optional schemas to store data.
+
+MongoDB is used to store and retrieve large amounts of data in real-time, and it is often used in situations where high performance and horizontal scaling are required.
+
+#### PostgreSQL
+
+[PostgreSQL](https://www.postgresql.org/) is a free and open-source relational database management system that is designed to be highly scalable, reliable, and powerful. It is known for its ability to handle large amounts of data and a high volume of queries. 
+
+PostgreSQL can handle complex data structures and relationships. It supports a wide range of data types, including arrays, JSON, and XML, and it has robust support for stored procedures, triggers, and views.
+
+#### MySQL
+
+[MySQL](https://www.mysql.com/) is an open-source relational database management system that is widely used in web applications and other software platforms. It is known for its reliability, simplicity, and performance. It is a powerful tool for storing and managing data in a structured and organized way.
+
+#### Kafka
+
+[Apache Kafka](https://kafka.apache.org/) is a distributed streaming platform, used for building real-time data pipelines and streaming applications. It is a publish-subscribe messaging system that allows for the creation of high-throughput and low-latency systems.
```

Comment:
We generate the Kafka client based on the [NestJS Kafka](https://docs.nestjs.com/microservices/kafka) that uses another Kafka client and not apache Kafka. This Kafka client is [KafkaJS](https://kafka.js.org/). 

```suggestion
[KafkaJS](https://kafka.js.org/) is a distributed streaming platform, used for building real-time data pipelines and streaming applications. It is a publish-subscribe messaging system that allows for the creation of high-throughput and low-latency systems.
```